lib/modules/manager/mix/extract.ts

```diff
@@ -60,11 +69,41 @@ export async function extractPackageFile(
             depName: app,
             currentValue: requirement,
             datasource: HexDatasource.id,
-            packageName: organization ? `${app}:${organization}` : app,
+            packageName: app,
           };
+
           if (requirement?.startsWith('==')) {
             dep.currentVersion = requirement.replace(regEx(/^==\s*/), '');
           }
+
+          if (organization) {
+            dep.packageName = `org:${organization}:${app}`;
```

Comment:
this is used by the hex datasource, so needs changes there too

https://github.com/renovatebot/renovate/blob/302718f38e54c6c7590f488d749fb4da8362f36f/lib/modules/datasource/hex/index.ts#L45

Reply:
That is in the other PR. You requested to split them so neither PR will work on its own

Reply:
then don't change package name in this PR, do the package name change in a separate refactor PR

Reply:
So neither PR is backwards compatible with the other. They are both dependent on the format change. If we don't change this here then your proposed refactor PR just pushes the change both need to make into a single PR that touches the datasource and manager.